build/gulpfile.extensions.js

```diff
@@ -274,7 +274,7 @@ gulp.task(packageLocalizationExtensionsTask);
 const compileLocalizationExtensionsBuildTask = task.define('compile-localization-extensions-build', task.series(
 	cleanExtensionsBuildTask,
 	compileExtensionsTask,
-	task.define('bundle-marketplace-extensions-build', () => ext.packageMarketplaceExtensionsStream(false).pipe(gulp.dest('.build'))),
+	task.define('bundle-marketplace-extensions-build', () => { ext.packageMarketplaceExtensionsStream(false, product.extensionsGallery?.serviceUrl).pipe(gulp.dest('.build')) }),
```

Comment:
add {{SQL CARBON EDIT}} tag?

@Charles-Gagnon I saw that in your commit, not all changes have the edit tag, was it a mistake or on purpose?

Reply:
All these tasks are in a SQL CARBON EDIT tag already so don't need to add another. Lines 244 & 285. Might be a good idea to move them out into a separate gulpfile though to make it easier to do merges...

My changes were to bring the task in line with what VS Code had at this point so doesn't need a SQL CARBON EDIT tag. (the only line that didn't - 234 - already had one from when Hai added it)